### ISSUE TITLE
r#keyword args get the 'r#' in introspection

### ISSUE THREAD

**Issue by zeenix** [z-galaxy/zbus#158]
In GitLab by @valpackett on Apr 18, 2021, 23:47

When an argument name is the same as a reserved keyword in Rust, one has to write the `r#` thing to be able to use it as an identifier:

```rust
    fn create_session(
        &self,
        uid: u32,
        pid: u32,
        service: String,
        r#type: String,
        …
```

Currently the zbus introspection generator includes the `r#` thing in the output though:

![Screen_2021-04-19-00_46_29](https://gitlab.freedesktop.org/dbus/zbus/uploads/b8c06f727c38a95c74832ddde537e058/Screen_2021-04-19-00_46_29.png)

**Comment by slightlyfaulty** [comment]
Just ran into this. Is there no fix planned?

A `type` argument name is pretty common, but not possible with zbus currently it seems.

**Comment by zeenix** [comment]
> Just ran into this. Is there no fix planned?

No. Just a reminder that this is a community project and hence if something is important to you, it's best to provide a PR for this (especially if there is an issue already and maintainers have expressed interest in fixing the issue).
